test/functional/test_runner.py

```diff
@@ -595,7 +594,7 @@ def run_tests(*, test_list, src_dir, build_dir, tmpdir, jobs=1, enable_coverage=
         print(f"{BOLD[1]}WARNING!{BOLD[0]} There may be insufficient free space in {tmpdir} to run the Bitcoin functional test suite. "
               f"Running the test suite with fewer than {min_space // (1024 * 1024)} MB of free space might cause tests to fail.")
 
-    tests_dir = build_dir + '/test/functional/'
+    tests_dir = f"{build_dir}/test/functional/"
```

Comment:
nit: This change seems irrelevant.

Reply:
I put it to a separate commit originally, @maflcko suggested I merge it into the same commit.
If you have strong feelings about it, I'll revert - but seems more idiomatic Python this way.